src/component/Footer.svelte

```diff
@@ -0,0 +1,153 @@
+<script>  
+    let srcMilk = '././img/image-gallery-milkbottles.jpg'; 
+    let srcOrange = '././img/image-gallery-orange.jpg';
+    let srcCone = '././img/image-gallery-cone.jpg';
+    let srcSugar = '././img/image-gallery-sugarcubes.jpg'; 
+
+    let srcLogo = '././img/sunnyside.svg'; 
+
+    let srcFacebook = '././img/icon-facebook.svg';
+    let srcInstagram = '././img/icon-instagram.svg';
+    let srcTwitter = '././img/icon-twitter.svg';
+    let srcPinterest = '././img/icon-pinterest.svg'; 
+</script>
+
+<footer>
+    <div class='footer-img'>
+        <img src={srcMilk} class='srcMilk' alt='MilkPhoto'>
+        <img src={srcOrange} class='srcOrange' alt='OrangePhoto'>
+        <img src={srcCone} class='srcCone' alt='ConePhoto'>
+        <img src={srcSugar} class='srcSugar' alt='SugarPhoto'>
+    </div>
+    <div class='footer-bottom'>
+        <div class='footer-content'>
+            <img src={srcLogo} class='logo' alt='SunnysideLogo'>
+            <ul class='nav-list'>
+                <li><a class='about' href="#about">About</a></li>
+                <li><a class='services' href="#services">Services</a></li>
+                <li><a class='projects' href="#projects">Projects</a></li>    
+            </ul>
+            <div class='icons'>
+                <img src={srcFacebook} class='srcFacebook' href="#" alt='FacebookIcon'>
+                <img src={srcInstagram} class='srcInstagram'href="#"  alt='InstagramIcon'>
+                <img src={srcTwitter} class='srcTwitter' href="#" alt='TwitterIcon'>
+                <img src={srcPinterest} class='srcPinterest'href="#"  alt='PinterestIcon'>
+            </div>
+        </div>
+    </div>
+</footer>
+
+<style>
+	footer {
+        position: absolute;
```

Comment:
absolute position cok gerekmedikce kullanmani tavsiye etmem. Responsive yapman gerektiginde zorluk yasarsin.

Reply:
Footer ve Header component'de absolute kullandım. Kullanmamamı daha önce de söylemiştin ama projeyi başlangıçta bu şekilde yaptığım için sonradan bu kısımları değiştirince karıştı ve düzeltemedim. O yüzde bu şekilde bıraktım. Bundan sonraki projelerde kullanmamaya çalışacağım.